Build failed with error Could not find resource 'src/resources/spotless/java.xml' if working dir is not root of ShardingSphere

## Bug Report

### Which version of ShardingSphere did you use?

master - e4d0557d5039dc1dce2c8d8c8fe9eb06b85aa1ac

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.

Failed:
```bash
% pwd
/home/wuweijie/projects/shardingsphere/sql-parser/dialect/mysql

% mvn clean install -q -DskipTests
```

This worked:
```bash
% pwd
/home/wuweijie/projects/shardingsphere

% mvn clean install -f sql-parser/dialect/mysql/pom.xml -q -DskipTests
```

### Expected behavior

Build specific modules successfully.

### Actual behavior

```
% pwd
/home/wuweijie/projects/shardingsphere/sql-parser/dialect/mysql
% mvn clean install -q -DskipTests
Running `/home/wuweijie/projects/shardingsphere/mvnw`...
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.22.1:apply (default) on project shardingsphere-sql-parser-mysql: Execution default of goal com.diffplug.spotless:spotless-maven-plugin:2.22.1:apply failed: Unable to locate file with path: src/resources/spotless/java.xml: Could not find resource 'src/resources/spotless/java.xml'. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

```

--- a/pom.xml
+++ b/pom.xml
@@ -856,10 +856,10 @@
                     <configuration>
                         <java>
                             <eclipse>
-                                <file>src/resources/spotless/java.xml</file>
+                                <file>${maven.multiModuleProjectDirectory}/src/resources/spotless/java.xml</file>
                             </eclipse>
                             <licenseHeader>
-                                <file>src/resources/spotless/copyright.txt</file>
+                                <file>${maven.multiModuleProjectDirectory}/src/resources/spotless/copyright.txt</file>
                             </licenseHeader>
                         </java>
                         <pom>

